### ISSUE TITLE
Chunk Loading Error

### ISSUE THREAD

**Issue by TheRealJayjayli** [chunky-dev/chunky#931]
Sorry for the undescriptive title, I don't know what to call this -

```
Exception in thread "Scene Manager" java.lang.ArrayIndexOutOfBoundsException: -779
	at se.llbit.math.PackedOctree.findSpace(PackedOctree.java:208)
	at se.llbit.math.PackedOctree.insertTempTree(PackedOctree.java:421)
	at se.llbit.math.PackedOctree.insertTempTree(PackedOctree.java:423)
	at se.llbit.math.PackedOctree.insertTempTree(PackedOctree.java:423)
	at se.llbit.math.PackedOctree.insertTempTree(PackedOctree.java:423)
	at se.llbit.math.PackedOctree.setCube(PackedOctree.java:508)
	at se.llbit.math.Octree.setCube(Octree.java:787)
	at se.llbit.chunky.renderer.scene.Scene.loadChunks(Scene.java:1198)
	at se.llbit.chunky.renderer.scene.SynchronousSceneManager.loadFreshChunks(SynchronousSceneManager.java:163)
	at se.llbit.chunky.renderer.scene.AsynchronousSceneManager.lambda$loadFreshChunks$2(AsynchronousSceneManager.java:139)
	at se.llbit.chunky.renderer.scene.AsynchronousSceneManager.run(AsynchronousSceneManager.java:81)
```

I also saw `Exception in thread "Scene Manager" java.lang.ArrayIndexOutOfBoundsException: -844` (`-844` instead of `-779`)

I had trouble finding the exact chunks - so I have attatched the region files, a small selection from IHOU2. I have also attatched some screenshots of what I see in the Render Preview:

![image](https://user-images.githubusercontent.com/52780721/117696483-40410080-b1b9-11eb-96b0-875cb8f15c37.png)

![image](https://cdn.discordapp.com/attachments/543090398671929344/841356921437028352/unknown.png)

**Comment by TheRealJayjayli** [comment]
Oops. Forgot the Region Files;
[ihou2.zip](https://github.com/chunky-dev/chunky/files/6453781/ihou2.zip)

**Comment by leMaik** [comment]
Fixed by #933
